onnxruntime/core/providers/qnn/builder/qnn_backend_manager.cc

```diff
@@ -128,6 +129,45 @@ Status QnnBackendManager::LoadQnnSystemLib() {
   return Status::OK();
 }
 
+void QnnLogging(const char* format,
+                QnnLog_Level_t level,
+                uint64_t timestamp,
+                va_list argument_parameter) {
+  ORT_UNUSED_PARAMETER(level);
```

Comment:
would also logging the QnnLog_Level_t value be useful?

Reply:
It's already included in Qnn log message for all Qnn log.